app/assets/javascripts/application.js

```diff
@@ -11,7 +11,7 @@
 //= require topbar
 //= require two-pane
 //= require vendor
-//= require about
+//= require statuses
```

Comment:
does it make sense to split them to separate files? I personally find it confusing